IBPSA/Fluid/FixedResistances/PlugFlowPipe.mo

```diff
@@ -274,8 +282,6 @@ The thermal capacity of the pipe wall is implemented as a mixing volume
 of the fluid in the pipe, of which the thermal capacity
 is equal to that of the pipe wall material.
 In addition, this mixing volume allows the hydraulic separation of subsequent pipes.
-Thanks to the vectorized implementation of the (design) outlet port,
-splits and junctions of pipes can be handled in a numerically efficient way.
 <br/>
 This mixing volume is not present in the
 <a href=\"modelica://IBPSA.Fluid.FixedResistances.BaseClasses.PlugFlowCore\">PlugFlowCore</a> model,
```

Comment:
I would delete that sentence as we agreed that the mixing volume was necessary to represent the effect of the pipe wall capacitance (not only for decoupling the DAE between multiple instances). 
Instead I would add something like:

> Note that in order to model a branched network it is recommended to use `IBPSA.Fluid.FixedResistances.Junction` at each junction and to configure that component with a state (see for instance
<a href=\"modelica://IBPSA.Fluid.FixedResistances.Validation.PlugFlowPipes.PlugFlowAIT\">
IBPSA.Fluid.FixedResistances.Validation.PlugFlowPipes.PlugFlowAIT</a>). 
This will avoid the numerical Jacobian that is otherwise created when the inlet ports of two instances of the plug flow model are connected together.

Reply:
I've reworded the whole documentation to incorporate part of the `PlugFlowCore` doc. Let me know if that works.